src/data/is_R_or_C/basic.lean

```diff
@@ -108,24 +108,24 @@ theorem ext_iff {z w : K} : z = w ↔ re z = re w ∧ im z = im w :=
 theorem ext {z w : K} (hre : re z = re w) (him : im z = im w) : z = w :=
 ext_iff.2 ⟨hre, him⟩
 
-@[norm_cast] lemma of_real_zero : ((0 : ℝ) : K) = 0 :=
-by rw [of_real_alg, zero_smul]
+@[norm_cast] lemma of_real_zero : ((0 : ℝ) : K) = 0 := algebra_map.coe_zero
 
-@[simp, is_R_or_C_simps] lemma zero_re' : re (0 : K) = (0 : ℝ) := re.map_zero
+@[is_R_or_C_simps] lemma zero_re' : re (0 : K) = (0 : ℝ) := map_zero re
 
 @[norm_cast] lemma of_real_one : ((1 : ℝ) : K) = 1 := map_one (algebra_map ℝ K)
 @[simp, is_R_or_C_simps] lemma one_re : re (1 : K) = 1 := by rw [← of_real_one, of_real_re]
 @[simp, is_R_or_C_simps] lemma one_im : im (1 : K) = 0 := by rw [← of_real_one, of_real_im]
 
 theorem of_real_injective : function.injective (coe : ℝ → K) := (algebra_map ℝ K).injective
-
-@[norm_cast] theorem of_real_inj {z w : ℝ} : (z : K) = (w : K) ↔ z = w :=
-algebra_map.coe_inj
+@[norm_cast] theorem of_real_inj {z w : ℝ} : (z : K) = (w : K) ↔ z = w := algebra_map.coe_inj
 
 @[simp, is_R_or_C_simps] lemma bit0_re (z : K) : re (bit0 z) = bit0 (re z) := map_bit0 _ _
+
 @[simp, is_R_or_C_simps] lemma bit1_re (z : K) : re (bit1 z) = bit1 (re z) :=
 by simp only [bit1, map_add, bit0_re, one_re]
+
 @[simp, is_R_or_C_simps] lemma bit0_im (z : K) : im (bit0 z) = bit0 (im z) := map_bit0 _ _
+
 @[simp, is_R_or_C_simps] lemma bit1_im (z : K) : im (bit1 z) = bit0 (im z) :=
```

Comment:
nit: could skip adding these spaces

Reply:
I prefer to have them because the `bit1` lemmas don't fit on 1 line.